src/Arrays/Programs_involving_Data_Sequences/Hard/Q1/MainTest.java

```diff
@@ -0,0 +1,65 @@
+package Arrays.Programs_involving_Data_Sequences.Hard.Q1;
+
+import global.BaseTest;
+import global.variables.*;
+import org.junit.jupiter.api.RepeatedTest;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class MainTest extends BaseTest {
+    int[] inputs = {0, 1, 0, 2, 3, 1, 0, 2, 0};
```

Comment:
is this variable named accurately? I don;t understand why this is an input? Is this supposed to be the correct counts?